tests/conftest.py

```diff
@@ -12,45 +13,20 @@
 from desc.vmec import VMECIO
 
 
+@pytest.fixture(scope="class", autouse=True)
+def clear_caches_before():
+    """Automatically run before each test to clear caches and reduce OOM issues."""
+    jax.clear_backends()
+    jax.clear_caches()
+
+
 @pytest.fixture(scope="session")
 def TmpDir(tmpdir_factory):
     """Create a temporary directory to store testing files."""
     dir_path = tmpdir_factory.mktemp("test_results")
     return dir_path
 
 
-@pytest.fixture(scope="session")
-def SOLOVEV_vac(tmpdir_factory):
```

Comment:
this isnt really a useful tests because vacuum tokamaks have ill defined flux surfaces